src/ontology/mondo-idranges.owl

```diff
@@ -185,5 +185,14 @@ Datatype: idrange:18
 Datatype: xsd:integer
 Datatype: rdf:PlainLiteral
```

Comment:
@paulacalipho could you please remove these 2 lines?
Datatype: xsd:integer
Datatype: rdf:PlainLiteral

These should only be at the very end of the file. (there was no way for you to know)